.github/workflows/windows.yml

```diff
@@ -473,6 +546,10 @@ jobs:
         timeout-minutes: 30
         shell: bash
         run: |
-          export TFLAGS='-j14 !TFTP !MQTT !WebSockets ${{ matrix.tflags }}'
-          ls bld/lib/*.dll >/dev/null 2>&1 && cp -f -p bld/lib/*.dll bld/tests/libtest/
+          export TFLAGS='-j14 ${{ matrix.tflags }}'
```

Comment:
```suggestion
          export TFLAGS='-j14 !TFTP !MQTT !WebSockets ${{ matrix.tflags }}'
```

Reply:
Why you don't want the other tests will run on the ci?

Reply:
They cause the tests to hang or fail, quite often. These exception were added after weeks of observing job runs and stabilizing them.

Reply:
ok, Thank you for the explanations!